RWSettings.php

```diff
@@ -380,6 +380,12 @@
 $wgGroupPermissions['tech']['gadgets-definition-create'] = true;
 $wgGroupPermissions['tech']['gadgets-definition-delete'] = true;
 
+#All users can view/restore deleted pages
+$wgGroupPermissions['*']['deletedhistory'] = true;
+$wgGroupPermissions['*']['deletedtext'] = true;
+$wgGroupPermissions['*']['undelete'] = true;
```

Comment:
The problem is that this also allows history merges. I will comment on the Saloon Bar.